TodoApp/Tests/TodoApp.TestContracts/Factories/ItemFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using TodoApp.Contracts.Models;
+
+namespace TodoApp.TestContracts.Factories
+{
+    public static class ItemFactory
+    {
+        public static Item CreateItem(string id, string text, DateTime? creationTime = null, DateTime? lastUpdateTime = null)
+            => CreateItem(Guid.Parse(id), text, creationTime, lastUpdateTime);
+
+        public static Item CreateItem(Guid id, string text, DateTime? creationTime = null, DateTime? lastUpdateTime = null)
+        {
+            creationTime = creationTime ?? DateTime.MinValue;
+            lastUpdateTime = lastUpdateTime ?? DateTime.MinValue;
```

Comment:
I wouldn't mind you inlining these 2 in favor of expression bodied function (highly optional).